modules/authentication/proc-authenticationg-with-the-guest-user-on-an-operator-based-installation.adoc

```diff
@@ -0,0 +1,28 @@
+:_mod-docs-content-type: PROCEDURE
+[id="authenticating-with-the-guest-user-on-an-operator-based-installation_{context}"]
+= Authenticating with the Guest user on an Operator-based installation
+
+After an Operator-based installation, you can configure {product-short} to log in as a Guest user and access {product-short} features.
+
+.Prerequisites
+* You link:[installed {product-short} by using the Operator].
+* You link:https://docs.redhat.com/en/documentation/red_hat_developer_hub/{product-version}/html-single/administration_guide_for_red_hat_developer_hub/index#proc-add-custom-app-config-file-ocp-operator_admin-rhdh[added a custom {product-short} application configuration], and have sufficient permissions to modify it.
+
+.Procedure
+* To enable the guest user in your {product-short} custom configuration, link:https://docs.redhat.com/en/documentation/red_hat_developer_hub/{product-version}/html-single/administration_guide_for_red_hat_developer_hub/index#proc-add-custom-app-config-file-ocp-operator_admin-rhdh[edit your {product-short} application configuration] with following content:
++
+.`app-config-rhdh.yaml` fragment
+[code,yaml]
```

Comment:
Or is [code, yaml] supposed to be used instead of [source, yaml] for all RHDH code samples?

Or is [code, yaml] specific to upstream? Or because this is a fragment and not a complete yaml example? I feel like I'm missing something so want to clarify.

Reply:
My mistake, again

```suggestion
[source,yaml]
```